.ci_support/linux_64_.yaml

```diff
@@ -1,3 +1,7 @@
+c_stdlib:
+- sysroot
+c_stdlib_version:
+- '2.12'
```

Comment:
Are you talking about this @beckermr?

AFAIU that would be a simple consequence of having those keys listed in smithy's `always_keep_keys`.

Reply:
Maybe we can not put it there since conda-build is fixed now?

Reply:
If we require conda-build >=24.3, then yes, it should be possible to remove.

I mainly followed `MACOSX_DEPLOYMENT_TARGET` being part of that list as well, and not managing to get things to work if I removed it from there. Perhaps the last refactor that also transforms `squishes_input_values` now makes this possible...

Reply:
Let's hold off for a while then. We can come back and cleanup later once the dust has settled.